### PR TITLE
Support Yield

### PR DESCRIPTION
**Pull request by bryantam**
Issue #206 
Current version of Swift does not support yield yet, so I was not able to find a way to generate a yield instruction

### CODE REVIEW

**Comment by bryantam** [comment]
Also, please wait until the PR into WALA is merged before merging this:
https://github.com/wala/WALA/pull/344

**Review by karimhamdanali — APPROVED** [review]
LGTM

**Pull request merged**
